### DISCUSSION TITLE
Goal 6: Ensure availability and sustainable management of water and sanitation for all

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#28] [General]
Goal 6: [Ensure availability and sustainable management of water and sanitation for all](https://github.com/Project-Resilience/platform/blob/main/goals/goal_06.md)

**Comment by danyoungday** [comment]
A potential dataset to be used: [https://environment.data.gov.uk/water-quality/view/download/new](https://environment.data.gov.uk/water-quality/view/download/new)

Compliance and monitoring data using both sensors and manual samples of regions in England.

Provided by the Environment Agency Department for Environment Food & Rural Affairs of the UK government.

There are features such as recorded chemical levels as well as human activity.

**Comment by danyoungday** [comment]
# EX-ACT for Irrigation

For the main discussion about this project see #35.

The EX-ACT tool by the FAO computes greenhouse gas emissions due to certain interventions. One such intervention is irrigation.

## Irrigation Parameters
For irrigation type and operation phase, multiple of these actions may be taken. For example, a user can create an intervention with 2 irrigation types and 3 operation phases.

- Irrigation type (surface without IRRS, surface with IRRS, solid set sprinkler, ...)
    - Hectares
        - With and without project

- Operation phase
    - Type of irrigation (center pivot sprinkler, trickle, ...)
    - Depth of well (m)
    - Source of energy (motor gasoline, waste oil, ...)
        - With and without project
    - Hectares
        - With and without project
    - Gross irrigation water requirement (mm/yr)
        - With and without project

Cost can be computed by gross irrigation water requirement and optimized for

**Comment by ofrancon** [comment]
This competition seems to fit SDG 6: 

1st AI for Drinking Water Chlorination Challenge @ IJCAI-2025
https://github.com/WaterFutures/AI-for-Drinking-Water-Chlorination-Challenge-IJCAI-25?tab=readme-ov-file#1st-ai-for-drinking-water-chlorination-challenge--ijcai-2025

> This challenge is about controlling the chlorination for mitigating and reacting to wastewater contamination events and other time-varying dynamics in a water distribution system simulation.
>
> The task is to control the injection of chlorine (Cl) at 5 locations, based on the readings of a few strategically placed hydraulic and chlorine sensors only.

The goals are to:
- minimize cost (amount for chlorine injected)
- minimize chlorine concentration 
- minimize infection risk

by deciding where, when, in which quantity to inject chlorine in the water distribution systems.
Looks like an interesting challenge!
